### PR TITLE
Create Description Component

### LINKED ISSUE #32: feat: add inspiration component
As a member i would like a text field where I submit a sentence or two about why I joined CFD.

We should limit the character length to 140 characters.

### LINKED ISSUE #39: feat: wire up cfd inspiration component
> reason for adding reason for joining CFD

### PR DESCRIPTION
**Pull request by jksmall0631**
This PR closes #32 closes #39 

#### What does this PR do?
-updated user schema to include description
-created description component
-created query to get current user id for mutation
-created mutation to update graph.cool
-pull out LoginAuth0 Apollo wrapper into variable for readability

### CODE REVIEW

File: src/Description.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  updateDB() {
+    const { description } = this.state;
+    const { id } = this.props.data.user
+    this.props.updateUser({ variables: { id, description } });
+  }
+
+  render() {
+    return (
+      <div>
```
Comment (dviramontes) [inline src/Description.js:38]: wondering if we can remove the empty wrapping `<div/>` ?

**Review by dviramontes — CHANGES_REQUESTED** [review]
Looks good 👍 , just the one comment/question
thanks @jksmall0631 !

File: src/Description.js (hunk 1 above)
Comment (jksmall0631) [reply src/Description.js:38]: I believe if you are returning multiple jsx elements you have to have a single wrapper...

Comment (dviramontes) [reply src/Description.js:38]: oh right! sorry i missed the other element thanks!

**Review by dviramontes — APPROVED** [review]
👍

**Pull request merged**